13. Roman to Integer/code.cpp

```diff
@@ -0,0 +1,66 @@
+//https://leetcode.com/problems/roman-to-integer/
+
+#include <iostream>
+
+ int romanToInt(std::string s) {
+    int sum = 0;
+    for (int i = 0; i <= s.length(); i++) {
+        if(s[i] == 'M') {
+            sum += 1000;
+        }
+        if (s[i] == 'D') {
+            sum += 500;
+        }
+        if (s[i] == 'C') {
+            if (s[i+1] == 'M') {
+                i++;
+                sum += 900;
+            }   else if(s[i+1] == 'D') {
+                i++;
+                sum += 400;
+            }   else {
+                sum += 100;
+            }
+        }
+        if (s[i] == 'L') {
+            sum += 50;
+        }
+        if (s[i] == 'X') {
+            if (s[i+1] == 'C') {
+                i++;
+                sum += 90;
+            }   else if (s[i+1] == 'L') {
+                    i++;
+                    sum += 40;
+                }   else sum += 10;
```

Comment:
Use else {
} even for 1-line expressions. Also check code formatting (something is wrong with braces.